CIP-XXXX/README.MD

```diff
@@ -0,0 +1,146 @@
+
+### Multi-Delegating
+- **Stake Key Derivation:** Begin by deriving the required number of stake keys needed for multi-delegation. While this derivation does not have to be sequential, it is recommended for clarity and orderly management. Ensure that the gap between any two consecutive active stake addresses does not exceed 20, in line with the [BIP-44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) address gap limit. 
+
+- **Key Registration:** Once derived, register these stake keys. This registration process remains consistent with current procedures, with the added context that all these new stake keys will be associated with a common payment key.
+
+- **Fund Distribution:** Distribute the funds across addresses formed by pairing the first  payment key with each of the newly created stake keys. This creates an association between all stake delegations. The details of how funds are distributed are specified by the user when creating the multi delegation transaction.
+
+Wallet software can then easily discover all stake keys and associated delegations either by sequential discovery or identifying addresses  containing the first payment credential. 
+
+### Updating the Multi-Delegation
+The dynamic nature of multi-delegation facilitates not just participation but also continuous optimization of delegations.
+
+#### Adding Pools
+To delegate to additional pools, the first step would involve deriving the necessary number of new stake keys. Then, make a transaction that:  registers the new stake keys, delegates and  distributes funds among addresses that pair the new stake keys with the first payment key.
+
+#### Removing Pools
+To remove a pool, the wallet first identifies all addresses associated with the stake key designated for removal. Then, make a transaction that
+Transfer all funds from these addresses
+withdraw all rewards from their associated reward accounts to one of the accounts that are still in delegation. If no other delegating accounts are left, revert the funds to addresses associated with the first payment key. 
+. Deregister stake keys
+
+**Note that all rewards calculated but not distributed will be lost.**
+
+#### Interoperability
+Live delegation portfolio might drift overtime due to:
+- Receiving funds to a single address 
+- Building transactions with software that doesn’t support multi-stake delegation (e.g. via dapp connector)
+
+In order to maintain a record of user’s delegation portfolio preferences , all actions pertaining to stake delegation, including funds transfers, stake key registration, and deregistration, must be executed within a single atomic transaction when performing an update. This transaction should also include the delegation portfolio metadata, using the reserved metadata label 6862 (hexadecimal 0x1ace), as outlined in [CIP-0017](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0017) preferably without any of the optional fields.
```

Comment:
Maybe make even more explicit that wallet apps must ensure that the chosen pools match between the CIP-17 JSON specification and the pools actually delegated to. … and that other wallet apps managing the same account are free to correct it if one misbehaves.

Should a new delegation certificate be included for *all* stakes and pools even if there is no change in that part of the multi-delegation? Advantage would be that this ensures that all stake keys have to sign the transaction partly attenuating @yHSJ's point on “Franken” addresses above. Disadvantage would be that it bloats the transaction and makes it more expensive unnecessarily.

Reply:
> Maybe make even more explicit that wallet apps must ensure that the chosen pools match between the CIP-17 JSON specification and the pools actually delegated to. … and that other wallet apps managing the same account are free to correct it if one misbehaves.

I like this idea

> … partly attenuating …

Since it doesn’t fully attenuate I’d vote in favour of not enforcing unnecessary certificates.

https://github.com/cardano-foundation/CIPs/pull/628#discussion_r1411743581